documentation/src/docs/asciidoc/user-guide/writing-tests.adoc

```diff
@@ -167,6 +167,30 @@ include::{testDir}/example/DisplayNameGeneratorDemo.java[tags=user_guide]
       '-- Year 2048 is a leap year. [OK]
 ```
 
+[[override-default-display-name-generator]]
```

Comment:
We never use generic anchors like that. Instead, all of our anchors are hierarchical according to the topic.

For example, `writing-tests-display-name-generator-default` would be more appropriate

Reply:
Thank you for the information. I will change accordingly.